src/mmapi.ts

```diff
@@ -0,0 +1,32 @@
+import superagent from 'superagent';
+import { Match } from './mmmatch';
+import { Player } from './mmplayer';
+
+export const getPlayerInfo = async (token: string) : Promise<Player|null> => {
```

Comment:
While not fully sure of your way to check for the returns, I would recommend using `Promise<Player?>` in place of `Promise<Player|null>` It's a bit more readable, and still allows for the usual checking of `if (returnValue)` to see if the value returned something that isn't null or undefined